reports/Iowa.md

```diff
@@ -186,15 +186,25 @@ id: ia-desmoines-8
 
 ## Iowa City
 
-### Police shoot tear gas at peaceful protestors who have their hands up | June 3rd
+### Iowa State Police use tear gas & flashbangs against crowd of civilians attempting to enter I-80 and later against Iowa City EMS. | June 3rd
 
-Iowa City Police shoot tear gas at peaceful protestors who have their hands up. Some protestors stumble and fall as they try to flee.
+State police shoot tear gas into, and deploy flashbangs against, a crowd of peaceful protesters attempting to enter I-80. Some protesters stumble and fall as they try to flee.
 
-tags: tear-gas, protestor
+Reportedly,  one protester suffered seizures as a result of a flashbang, and an ambulance was called to the scene.
+
+Police continued to deploy tear gas and flashbangs in the area where Iowa city EMT's were working (see flashing white lights in cloud of gas, 5th link). Unclear if they were aware of EMS presence.
+
+According to the mayor, Iowa State Patrol was in command of the scene (though some Iowa City Police were present) and it was determined that letting the protest onto I-80 was too dangerous.
+
+tags: tear-gas, protestor, flash-bang, ems, emt, interstate, i-80
```

Comment:
Check back on the possible tags [here](https://github.com/2020PB/police-brutality/blob/master/docs/possible_tags.md). I would add flashbangs (no hyphen) and "medic," which is our catchall for any kind of medical services. And Interstate/i-80 aren't really in scope as of now for tags.